Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,33 @@
 import React from 'react';
-import './Header.css';
+// import './header.css';
+
+const HeaderContent = () => {
+    return (
+        <>
+            <div className="headerContent">
+                Let's learn React by building simple interfaces with components.
+                Don't try to overthink it, just keep it simple and have fun. Once you feel
+                comfortable using components you are well on your way to mastering React!
+            </div >
+
+        </>
+     )
+};
+
+export default HeaderContent;
+
+
+// Const HeaderContent = () => {
```

Comment:
I know this is commented out, but I wanted to mention what could have possibly prevented this code from running. 

`Const` and `const` are two completely different things to js. `const` is a reserved keyword. `Const` is a string, could be a variable, or something else. 

The component would run if you did this:
```suggestion
// const HeaderContent = () => {
```